sway/ipc-json.c

```diff
@@ -199,7 +199,7 @@ static void ipc_json_describe_output(struct sway_output *output,
 
 	struct sway_workspace *ws = output_get_active_workspace(output);
 	json_object_object_add(object, "current_workspace",
-			json_object_new_string(ws->name));
+			ws ? json_object_new_string(ws->name) : NULL);
```

Comment:
This should probably be an assertion too

Reply:
Changed